src/components/Annotation.js

```diff
@@ -1,14 +1,30 @@
-import React from "react"
+// @flow
+import * as React from "react"
 import AnnotationLabel from "react-annotation/lib/Types/AnnotationLabel"
 
-import PropTypes from "prop-types"
+type Props = {
+  noteData: {
+    eventListeners: Object,
+    type: ?function,
```

Comment:
This `type` attribute could be much more rigorously defined so that it accepts the props it is fed below. I left it alone though to get this bootstrapped.